products/sle/main.pm

```diff
@@ -863,8 +864,8 @@ elsif (get_var("VIRT_AUTOTEST")) {
         loadtest "virt_autotest/install_package";
         loadtest "virt_autotest/update_package";
         # Skip reset_partition for s390x due to there just be 42Gib disk space for each s390x LPAR
-        loadtest "virt_autotest/reset_partition" if (!is_s390x and get_var('VIRT_PRJ1_GUEST_INSTALL'));
-        loadtest "virt_autotest/reboot_and_wait_up_normal" if (!get_var('AUTOYAST') && get_var('REPO_0_TO_INSTALL'));
+        loadtest "virt_autotest/reset_partition" if (is_x86_64 and get_var('VIRT_PRJ1_GUEST_INSTALL') and !get_var('LTSS'));
```

Comment:
```suggestion
        loadtest "virt_autotest/reset_partition" if (is_x86_64 && get_var('VIRT_PRJ1_GUEST_INSTALL') && !get_var('LTSS'));
```

Reply:
May I know why "and" are suggest to be replaced with "&&"?

Reply:
I don't think there is any difference between `and` / `&&`, IMO, based on the code style, they should keep in the same style :). anyway, it is not a big deal.

Reply:
Yes, more tidy. thanks.